core/src/main/resources/lib/form/hetero-list.jelly

```diff
@@ -123,10 +127,22 @@ THE SOFTWARE.
       <j:set var="instance" value="${null}" />
       <j:set var="descriptors" value="${h.filterDescriptors(it,attrs.descriptors)}" />
```

Comment:
@varyvol: `h` is used here…

Reply:
Yes, I know it's used here, but why does it need to be added to the other line?